tests/tree/test_tree_operators.py

```diff
@@ -5,6 +5,7 @@
 import discretize
 
 MESHTYPES = ['uniformTree'] #['randomTree', 'uniformTree']
```

Comment:
I take it this does not work for a `randomTree`?

Reply:
It does, we weren't previously running tests on a randomTree, but I have put this back in.